src/ui/components/AppHeader/app-header.scss

```diff
@@ -79,35 +78,32 @@
       width: 0;
       height: 0;
       bottom: 100%;
-      left: 1.5em; // offset should move with padding of parent
-      border: .375rem solid transparent;
+      left: 1.5em;
+      //offsetshouldmovewithpaddingofparentborder: .375rem solid transparent;
```

Comment:
Looks like the comment was moved into the next property and commented out the border xD

Reply:
Thanks for pointing out, will address before merge. Thanks!